builtin/provisioners/chef/linux_provisioner.go

```diff
@@ -10,10 +10,27 @@ import (
 )
 
 const (
-	chmod      = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
+	chmodLinux = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
+	chmodAix   = "find %[1]s ! -path %[1]s -prune -type f -exec /bin/chmod %[2]d {} +"
 	installURL = "https://omnitruck.chef.io/install.sh"
 )
 
+func (p *provisioner) getChmodCommand() string {
+	if p.OSType == "aix" {
+		return chmodAix
+	} else {
+		return chmodLinux
+	}
+}
+
+func (p *provisioner) getRootGroup() string {
```

Comment:
In Go getters are normally not prefixed by `set`:

```suggestion
func (p *provisioner) rootGroup() string {
```

Reply:
That's good to know (Go newbie here). I think with the fix below to create a `chownAIX` and `chownLinux` command constant, this function will go away.